compiler/src/skipError.sk

```diff
@@ -91,8 +91,12 @@ fun throwFlattenedErrors(errors: Vector<Vector<Error>>): void {
 
 fun printErrorsAndExit<T>(
   errors: Array<SkipErrorException>,
-  get_file: String -> (String, String) = filename ->
-    (filename, FileSystem.readTextFile(filename)),
+  get_file: String -> (String, String) = filename -> {
+    if (filename.contains(":")) {
```

Comment:
It's not common or a good idea, but `:` is legal in unix file/directory names... this could lead to some weird behaviours if we encounter that.  Could we match something a little stricter for this case?

Reply:
My bad, it's up to date now.

Reply:
Yes, the current encoding of `(?lib, path)` as `lib:path` is a bit fragile (though it shouldn't be an issue as long as we don't use `:` in file names). Doing this correctly is on my TODO list but it requires quite a few changes in the compiler.